src/pages/ConciergePage.js

```diff
@@ -26,7 +25,7 @@ const propTypes = {
  */
 const ConciergePage = (props) => {
     if (_.has(props.session, 'authToken')) {
```

Comment:
Do we still need to check for this?

Reply:
Yeah, not part of your changes but I am curious why it is here (maybe when we link to this page somewhere?)

Reply:
Yeah it looks like we link directly there from oldDot sometimes. There are at least 2 places it's directly routed/rendered. Must just be a fallback